### ISSUE TITLE
Nuget Package not working

### ISSUE THREAD

**Issue by anf** [radimitrov/CSharpShellApp#61]
Hi there,
I installed a few nuget packages from the nuget app and all of them installed except one: Microsoft.Extensions.
For some reason it was not registering it. Idek how is that possible. I tried both Global and local package but nothing seems to work. Yes I did tick it in the manager, I also used other packages. Any idea what's going on? Here is a snip of it:![Screenshot_20200727-074119.png](https://user-images.githubusercontent.com/68814933/88498488-b256a900-cfdc-11ea-83c0-81e0c42083ec.png)

**Comment by radimitrov** [comment]
Strange. Maybe it is incompatable with Android? 
Which Microsoft.Extensions package? There must be hundreds.

**Comment by ANF-Studios** [comment]
> Strange. Maybe it is incompatable with Android? 
I don't think that's the case because I found something really interesting..

> Which Microsoft.Extensions package? There must be hundreds.
Well, basically any (in my case Microsoft.Extensions.DependencyInjection and .DependencyInjection.Abstraction).

And with that, I found something with some of my own resrearch. First I want to show you what happened from the beginning: ![Screenshot_20200731-063119.png](https://user-images.githubusercontent.com/68814933/88991245-98c5a200-d2f9-11ea-8683-3f6fc5cea9fd.png)
I had installed all the required packages (also with some more packages (which did work)). Now this is what I found in the code: ![Screenshot_20200731-063039.png](https://user-images.githubusercontent.com/68814933/88991363-e93cff80-d2f9-11ea-9a93-61b73c2d7699.png) Hmm, well I found my way into the Default Assemblies (Global Assemblies References) and ticked whatever I needed. ![Screenshot_20200731-063227.png](https://user-images.githubusercontent.com/68814933/88991454-286b5080-d2fa-11ea-97f9-d60bdab0e114.png) And I found that it finally detected it:  ![Screenshot_20200731-063309.png](https://user-images.githubusercontent.com/68814933/88991560-6ec0af80-d2fa-11ea-9562-76c4b048d330.png) And voila, but my theory is that if I install a Nuget Package which already exists from the Default Assemblies, it'll not work. Please further investigate.
In summary, the issue is partially closed, but it needs to be fixed.

**Comment by radimitrov** [comment]
No, I don't think it is related to the default assemblies.
But that does confirm that I'll have to take a detailed look what goes wrong at runtime since the problem clearly isn't loading the assembly. I hope it isn't a weird path length issue.
